ros_gz_sim_demos/config/full.yaml

```diff
@@ -0,0 +1,18 @@
+# Full set of configurations
+- ros_topic_name: "ros_chatter"
+  gz_topic_name: "gz_chatter"
+  ros_type_name: "std_msgs/msg/String"
+  gz_type_name: "ignition.msgs.StringMsg"
+  subscriber_queue: 5
+  publisher_queue: 6
+  lazy: true
+  direction: ROS_TO_GZ
+
+- ros_topic_name: "ros_chatter"
+  gz_topic_name: "gz_chatter"
+  ros_type_name: "std_msgs/msg/String"
+  gz_type_name: "ignition.msgs.StringMsg"
```

Comment:
I believe we can use `gz.msgs` for the `gz_type_name`.

Reply:
Changed in https://github.com/gazebosim/ros_gz/pull/511.